### PR TITLE
Add Kotlin version table

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/compatibility.adoc [hunk 1]
```diff
@@ -48,6 +48,27 @@ Gradle is tested with Kotlin 1.3.72 through 1.6.21.
 
 Gradle plugins written in Kotlin target Kotlin 1.4 for compatibility with Gradle and Kotlin DSL build scripts, even though the embedded Kotlin runtime is Kotlin 1.6.
 
+.Embedded Kotlin version
```
Comment (eskatos) [inline subprojects/docs/src/docs/userguide/compatibility.adoc:51]: I would put the Gradle version in the first column

**Review by eskatos — COMMENTED** [review]
Yeah! Thanks for doing this

File: subprojects/docs/src/docs/userguide/compatibility.adoc (hunk 1 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/userguide/compatibility.adoc:51]: I did it like that to be aligned with the java table, but you are right, it makes more sense.

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Review by eskatos — APPROVED** [review]

**Pull request merged**